internal/controller/prompt.go

```diff
@@ -54,5 +55,6 @@ func TeleportPrompt(envName []string, envDetails map[string]string) string {
 		utils.TerminateWithCustomError("Aborted!")
 	}
 
-	return envDetails[selectedEnv]
+	return envDetails[selectedEnv].EnvId
```

Comment:
comment - is it possible that selected env = invalid? If not, then no need to check.

But if so, I would do 

if envDetail, ok := envDetails[selectedEnv]; ok { 
... return good shit.
}

return error.

Reply:
SelectedID won't ever be invalid unless, someone goes up til the env selection menu and then deletes one of his env. This is like the edge case of edge cases. But if needed can add a check for it